src/main/groovy/net/foragerr/jmeter/gradle/plugins/worker/JMeterRunner.groovy

```diff
@@ -48,12 +48,6 @@ class JMeterRunner {
         argumentsList.toArray(new String[argumentsList.size()])
     }
 
-    /**
```

Comment:
Undo this comment delete and I'll approve your PR. Everything else looks perfect, Thanks for your contribution!